app/js/app/controllers/SearchControllers.js

```diff
@@ -89,7 +90,13 @@ define([], function() {
         });
 
         $scope.$watch('response.results', function(results) {
-            if ($scope.response) {
+            if ($scope.response && results) {
+                if (!(shortcutResponse === undefined || shortcutResponse.length == 0)) {
+                    let shortcutResults = shortcutResponse;
```

Comment:
Why do you need `shortcutResults` here, if you're going to set `shortcutResponse` to an empty list below anyway? I don't  even think `.concat()` modifies either list anyway.

Reply:
I added this as an interim and didn't remove it, thanks!